content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,150 @@
+# Live.log_plot()
+
+```py
+def log_plot(self, name: str, labels, predictions, **kwargs):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+y_true = [0, 0, 1, 1]
+y_pred = [1, 0, 1, 0]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("roc", y_true, y_score)
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+## Description
+
+Generates a
+[scikit learn plot](https://scikit-learn.org/stable/visualizations.html) and
+saves the result in `{path}/plots/{name}.json`.
+
+💡 The generated `{path}/plots/{name}.json` can be visualized with `dvc plots`.
+
+### Step updates
+
+⚠️ `Live.log_plot()` can't be currently used in scripts where the step is being
+updated ( with `Live.next_step()` or `Live.set_step()`).
+
+## Supported plots
+
+`name` must be one of the supported plots:
```

Comment:
I would either list them all at the top with links, or make them tabs, collapsible, or some other way to see them all quickly. Especially if we list in alphabetical order, having to scroll all the way down to find ROC, for example, seems inconvenient.

Reply:
Maybe i'm like a child with a new toy but I added `tabs` and I really like how it looks

Reply:
Good thinking on tabs BUT I'm not sure we want to use them in this way. Are we OK with hiding all the details for everything except `calibration` behind a click? We [discussed](https://github.com/iterative/dvc.org/pull/3178#issuecomment-1034120050) this earlier with @iesahin and @shcheklein , and decided this is not the core function of tabs.

Reply:
Then again looking at the [published version](https://dvc.org/doc/dvclive/api-reference/live/log_plot#supported-plots) I do appreciate that the tabs toggle prevents the page to become much much longer. Maybe a solution is to add a bullet list with each type and short definition and then the tabs toggle for details underneath?